Tests/BotTests/MergeService/DispatchServiceTests.swift

```diff
@@ -33,6 +33,7 @@ class DispatchServiceTests: XCTestCase {
             ],
             when: { service, scheduler in
                 scheduler.advance()
+                scheduler.advance()
```

Comment:
What's happening here exactly? This is very strange. cc @AliSoftware

Reply:
I suppose this is somehow related to the test fixes on Linux, but as they didn't really solved the issue maybe we should remove these changes from this PR?

Reply:
Yeah we should remove that. Was a (lame) attempt to see if that was the reason for the test failure. But we'll try to handle the debugging of that in #47 so we should indeed remove that in this PR.
```suggestion
```

Reply:
Ok, I'll revert your commit then @AliSoftware I suppose it's already in another branch dedicated to the tests fix